### PR TITLE
Copy structure-change from Core Plugwise

### PR DESCRIPTION
**Pull request by bouwew**

### CODE REVIEW

File: custom_components/plugwise/__init__.py [hunk 1]
```diff
@@ -48,161 +21,12 @@ async def async_setup(hass: HomeAssistant, config: dict):
...
-    """Handle options update."""
-    coordinator = hass.data[DOMAIN][entry.entry_id][COORDINATOR]
-    coordinator.update_interval = timedelta(
-        seconds=entry.options.get(CONF_SCAN_INTERVAL)
-    )
-
+    """Set up Plugwise components from a config entry."""
+    if entry.data.get(CONF_HOST):
+        return await async_setup_entry_gw(hass, entry)
+    # PLACEHOLDER USB entry setup
```
Comment (CoMPaTech) [inline custom_components/plugwise/__init__.py:28]: True if we don't want to include #117 just yet (which was the goal, right)

File: custom_components/plugwise/strings.json [hunk 2]
```diff
@@ -13,6 +13,13 @@
     "flow_title": "Smile: {name}",
     "step": {
       "user": {
+        "title": "Plugwise type",
+        "description": "Product:",
+        "data": {
+          "flow_type": "Connection type"
```
Comment (CoMPaTech) [inline custom_components/plugwise/strings.json:19]: This bit might be where the 'flow_action' or thing from #117 came from, but it didn't reproduce for me in Core

**Review by CoMPaTech — APPROVED** [review]
LGTM

File: custom_components/plugwise/__init__.py (hunk 1 above)
Comment (bouwew) [reply custom_components/plugwise/__init__.py:28]: Yes, that's the idea, first, implement the structure-change from Core. Next step, implement #117.

File: custom_components/plugwise/strings.json (hunk 2 above)
Comment (bouwew) [reply custom_components/plugwise/strings.json:19]: I've tested with this code present, adding an Adam, both via discovery and manual adding, works as it did before the change.

**Pull request merged**